vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        """Finds the contours from a mask image."""
</pre><p>This line could be a bit more descriptive, such as: "Finds the contours from a mask image, and returns it as a ...."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Removed since I feel the code is descriptive enough anyway.</p>